lib/src/gestures/gestures.dart

```diff
@@ -30,37 +29,18 @@ abstract class MapGestureMixin extends State<FlutterMap>
     if (pointerSignal is PointerScrollEvent &&
         mapState.options.enableScrollWheel) {
       if (pointerSignal.scrollDelta.dy != 0) {
-        _debounce(() {
-          // Check whether the mouse is scrolled down and calculate new zoom level
-          final delta = pointerSignal.scrollDelta.dy > 0 ? -0.25 : 0.25;
-          print(pointerSignal.scrollDelta.dy == 0);
-          final zoom = delta > 0
-              ? min(mapState.options.maxZoom ?? 19, mapState.zoom + delta)
-              : max(mapState.options.minZoom ?? 0, mapState.zoom + delta);
-          print('zoom: $zoom');
-          final newZoom = mapState.fitZoomToBounds(zoom);
-          // Move the map to the new zoom level
-          mapState.move(mapState.center, newZoom,
-              source: MapEventSource.custom);
-        });
+        // Check whether the mouse is scrolled down and calculate new zoom level
+        final delta = pointerSignal.scrollDelta.dy * (-0.01);
+        final zoom = delta > 0
+            ? min(mapState.options.maxZoom ?? double.infinity, mapState.zoom + delta)
```

Comment:
@JaffaKetchup there was no `?? 19` other check, so it was rather a mistake (copied from previous behaviour). Just changed this to coerce between `maxZoom` and infinity instead.